main/docs/authenticate/single-sign-on/native-to-web/configure-mobile-to-web-payment-flows.mdx

```diff
@@ -590,4 +590,4 @@ You have created a seamless experience where a mobile native application user ca
 ## Next Steps
 
 * Explore more configuration options for Native to Web SSO:Dive deeper into session lifetime, <Tooltip tip="Refresh Token: Token used to obtain a renewed Access Token without forcing users to log in again." cta="View Glossary" href="/docs/glossary?term=refresh+token">refresh token</Tooltip> rotation, device binding, and cascade revocation in the [Native to Web SSO documentation](/docs/authenticate/single-sign-on/native-to-web).
-* Customize the post-login experience with Progressive Profiling:Use Auth0’s [Progressive Profile Form](/docs/customize/forms/configure-progressive-profile-form) to collect additional user data after login — such as plan preferences, address, or payment intent — before showing subscription options.
\ No newline at end of file
+* Customize the post-login experience with Progressive Profiling:Use Auth0’s [Progressive Profile Form](/docs/customize/forms/configure-progressive-profile-form) to collect additional user data after login — such as plan preferences, address, or payment intent — before showing subscription options.
```

Comment:
Could use a space between `Profiling:Use`, so `Profiling: Use` the way it is in the current doc.